src/Compiler/Service/service.fs

```diff
@@ -402,9 +405,7 @@ type BackgroundCompiler
         | Some getBuilder ->
             node {
                 match! getBuilder with
-                | builderOpt, creationDiags when builderOpt.IsNone || not builderOpt.Value.IsReferencesInvalidated ->
-                    Logger.Log LogCompilerFunctionId.Service_IncrementalBuildersCache_GettingCache
-                    return builderOpt, creationDiags
+                | builderOpt, creationDiags when builderOpt.IsNone || not builderOpt.Value.IsReferencesInvalidated -> return builderOpt, creationDiags
```

Comment:
Is this formatting Fantomas' default? 🤔

Reply:
I applied a full Fantomas check on the file. It should be deciding inline/linebreak depending on the width.